Cargo.toml

```diff
@@ -52,6 +53,11 @@ walkdir = "2"
 async-recursion = "1.0.5"
 thiserror = "1.0.49"
 miette = { version = "5.10.0", features = ["fancy"] }
+protobuf = { version = "3.3.0", optional = true }
+protobuf-parse = { version = "3.3.0", optional = true }
+diff-struct = { version = "0.5.3", optional = true }
+derive_more = "0.99.17"
```

Comment:
Is this required? If its absolutely necessary please make it local to the validation. But in the best case we could remove it

Reply:
I'll remove it, this will add ~15 lines of code tho.